subsys/bluetooth/mesh/Kconfig

```diff
@@ -21,6 +21,13 @@ if BT_MESH
 config BT_MESH_PROV
 	bool
 
+config BT_MESH_NODE
+	bool "Supporting device provisioning"
+	select BT_MESH_PROV
+	default y
+	help
+	  Enable this option to allow the device supporting provisioning.
```

Comment:
If I remember correctly, `BT_MESH_PROV` (which indicates that we want support for the generic provisioning protocol) has so far been selected indirectly by choosing which provisioning bearer you want. Saying you want `BT_MESH_PROVISIONEE` doesn't tell you anything about provisioning bearers, so I'm not sure this is the right thing. So unless we want to break the semantics on the way that you choose mesh provisioning Kconfig options it seems to me like this `BT_MESH_PROVISIONEE` should `depend on BT_MESH_PROV` instead of selecting it.

Reply:
Sounds good to me